.redocly.yaml

```diff
@@ -1,9 +1,8 @@
-lint:
```

Comment:
The redoc docs mention that `lint` was replaced with some new thing, then that thing was also deprecated. Just removing this top level object and shifting everything down "just worked".